init.rb

```diff
@@ -1,12 +1,14 @@
-require 'redmine'
+# Global Hooks
+require File.expand_path('../lib/redmine_gtt/hooks/view_layouts_base_html_head_hook', __FILE__)
+require File.expand_path('../lib/redmine_gtt/view_hooks', __FILE__)
 
 Redmine::Plugin.register :redmine_gtt do
   name 'Redmine GTT plugin'
   author 'Georepublic'
   author_url 'https://github.com/georepublic'
   url 'https://github.com/gtt-project/redmine_gtt'
   description 'Adds location-based task management and maps'
-  version '3.0.0'
+  version '3.0.0' # TODO: 4.0.0?
```

Comment:
Need to confirm whether major version up is necessary.

Reply:
We need major version up and write some comment of version into README.

Reply:
Okay, thanks for confirmation.
Then, I will update the major version with README.
(But, please wait for a while until checking other plugins (about next Monday), because sometimes plugins loading order is quite sensitive...)

Reply:
@smellman @dkastl  
I upgrade the major version and I wrote the `4.0.0` version info at `Version History` section in this PR.

Also, I updated some minimum version requirements from the following related links:
- PostgreSQL 9.6 => 10: https://www.postgresql.org/support/versioning/
- PostGIS 2.4 => 2.5: https://trac.osgeo.org/postgis/wiki/UsersWikiPostgreSQLPostGIS
- Redmine 4.0 => 4.2: https://www.redmine.org/issues/33290#note-10